### PR TITLE
Usage Puppet Providers Mock

### PR DESCRIPTION
**Pull request by Soumi7**
Updates Puppet Providers - Mock section in Puppet Providers

References issue #890

Screenshots :

image

image

image

image

image

image

image

image

### CODE REVIEW

**Review by shraddhavp — APPROVED** [review]
LGTM

File: docusaurus/docs/puppet-providers/mock.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ sidebar_label: Mock
 
 ## Features
 
-1. Help to test Wechaty framework with a mock puppet
-1. Using as a starter template for write your own puppet provider
+1. Helps to test Wechaty framework with a mock puppet.
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/mock.md:18]:
```suggestion
1. Help to test Wechaty framework with a mock puppet.
```
* imperative sentence

File: docusaurus/docs/puppet-providers/mock.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ sidebar_label: Mock
...
+2. Can be used as a starter template for writing your own puppet provider.
 
 ## Usage
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/mock.md:22]: Run `wechaty-puppet-mock`.

File: docusaurus/docs/puppet-providers/mock.md [hunk 3]
```diff
@@ -83,7 +83,7 @@ wechaty.start()
 // The Mocker will start perform the SimpleEnvironment...
 ```
 
-See: [SimpleEnvironment](https://github.com/wechaty/wechaty-puppet-mock/blob/master/src/mock/environment.ts)
+See: [SimpleEnvironment](https://github.com/wechaty/wechaty-puppet-mock/blob/master/src/mock/environment.ts).
 
 ## API Reference
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/mock.md:91]: Please add explanation of the code snippet here.

File: docusaurus/docs/puppet-providers/mock.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ wechaty.start()
 // The Mocker will start perform the SimpleEnvironment...
 ```
```
Comment (proudofsimin) [inline docusaurus/docs/puppet-providers/mock.md:86]: (not sure why I can't insert in between the title "Mocker and environment" and the code snippet) <-Please change the title to this format as well.

Please add explanation of the code snippet.

**Review by proudofsimin — CHANGES_REQUESTED** [review]
Always remember that you should never leave code snippet alone without explanations. Please describe what you are doing in the codes. Examples could see [documents](https://github.com/wechaty/wechaty.js.org/pull/981) from How-to guides team, or from [Javascript](https://javascript.info/date).

**Comment by Soumi7** [comment]
Updated screenshots :

image

image

image

**Comment by Soumi7** [comment]
@proudofsimin No more information other than the titles is mentioned anywhere, hence I think we will have to request the mentors for more info about the code snippets. @shraddhavp  what do you think?

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Comment by Rohitesh-Kumar-Jain** [comment]
@proudofsimin I would be glad if you review changes once again : )

**Comment by Soumi7** [comment]
@proudofsimin as Huan has already approved this, could you please approve this too? There is nothing more I can add in this.

**Review by proudofsimin — APPROVED** [review]

**Pull request merged**